### PR TITLE
Release v3.0.3

### PR DESCRIPTION
**Pull request by parkr**
Bug fixes for a few issues noticed after last week's upgrade to Jekyll 3 that GitHub made.
Thanks!

/cc @jekyll/gh-pages @jekyll/core

### CODE REVIEW

**Comment by benbalter** [comment]
:sparkling_heart:

File: site/_posts/2016-02-08-jekyll-3-0-3-released.markdown [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: news_item
+title: 'Jekyll 3.0.3 Released'
+date: 2016-02-08 10:39:08 -0800
+author: parkr
+version: 3.0.3
+categories: [release]
+---
+
+[GitHub Pages upgraded to Jekyll 3.0.2][1] last week and there has been a
```
Comment (benbalter) [inline site/_posts/2016-02-08-jekyll-3-0-3-released.markdown]: Fine as is, but:

1. *some* users had *some* bugs feels a bit weasily
2. Joyous reception? Show, don't tell. If we can't show, I'd cut it.
3. What if we make this a positive, instead of a negative? It's not that we :ship:'d with a bunch of bugs, but the cool thing about having a test bed of 1M+ sites is that you can quickly and easily surface unique bugs and edge cases.

File: site/latest_version.txt [hunk 2]
```diff
@@ -1 +1 @@
-3.0.1
+3.0.2
```
Comment (benbalter) [inline site/latest_version.txt]: Should this be .03?

File: site/_posts/2016-02-08-jekyll-3-0-3-released.markdown (hunk 1 above)
Comment (parkr) [inline site/_posts/2016-02-08-jekyll-3-0-3-released.markdown]: Thanks! 29a5a3c303c64abf90100cb6928c096c1cf8dc63 look ok?

Comment (benbalter) [inline site/_posts/2016-02-08-jekyll-3-0-3-released.markdown]: :sparkles:
